internal/tiltd/tiltd.go

```diff
@@ -42,3 +42,7 @@ type Cmd struct {
 func (c Cmd) EntrypointStr() string {
 	return fmt.Sprintf("ENTRYPOINT [\"%s\"]", strings.Join(c.Argv, "\", \""))
 }
+
+func ToShellCmd(cmd string) Cmd {
```

Comment:
Would it make sense for this to live on the Cmd struct?

Reply:
the thing i'm trying to avoid is the boilerplate of wrapping something in
```
myCmd := Cmd{Argv: []string{"blah"}}
```
and if this func lived on the `Cmd` struct I'd still have to do that :-/